src/main/java/com/microsoft/sqlserver/jdbc/osgi/Activator.java

```diff
@@ -38,6 +38,12 @@ public void start(BundleContext context) throws Exception {
         properties.put(DataSourceFactory.OSGI_JDBC_DRIVER_NAME, "Microsoft JDBC Driver for SQL Server");
         properties.put(DataSourceFactory.OSGI_JDBC_DRIVER_VERSION,
                 driver.getMajorVersion() + "." + driver.getMinorVersion());
+        properties.put(DataSourceFactory.OSGI_JDBC_CAPABILITY, new String[] {
+                DataSourceFactory.OSGI_JDBC_CAPABILITY_DRIVER,
+                DataSourceFactory.OSGI_JDBC_CAPABILITY_DATASOURCE,
+                DataSourceFactory.OSGI_JDBC_CAPABILITY_CONNECTIONPOOLDATASOURCE,
+                DataSourceFactory.OSGI_JDBC_CAPABILITY_XADATASOURCE
+        });
```

Comment:
I'm not familiar with OSGI and how these properties are used, but we do have a test that check for properties can we add these to the test?  https://github.com/microsoft/mssql-jdbc/blob/main/src/test/java/com/microsoft/sqlserver/jdbc/osgi/DataFactoryTest.java

Reply:
Done!